<ui version="4.0" >
 <class>genericDeviceOptions</class>
 <widget class="QWidget" name="genericDeviceOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>545</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>11</number>
   </property>
   <property name="topMargin" >
    <number>11</number>
   </property>
   <property name="rightMargin" >
    <number>11</number>
   </property>
   <property name="bottomMargin" >
    <number>11</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QCheckBox" name="kcfg_status_poll" >
     <property name="text" >
      <string>C&amp;heck for phone status/incoming calls</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>11</number>
     </property>
     <property name="topMargin" >
      <number>11</number>
     </property>
     <property name="rightMargin" >
      <number>11</number>
     </property>
     <property name="bottomMargin" >
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="poll2Label" >
       <property name="text" >
        <string>Time for check (seconds):</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KDoubleNumInput" native="1" name="kcfg_status_pollTimeout" />
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>11</number>
     </property>
     <property name="topMargin" >
      <number>11</number>
     </property>
     <property name="rightMargin" >
      <number>11</number>
     </property>
     <property name="bottomMargin" >
      <number>11</number>
     </property>
     <item>
      <widget class="QCheckBox" name="kcfg_smsPoll" >
       <property name="text" >
        <string>Poll automaticall&amp;y for sms</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>11</number>
       </property>
       <property name="topMargin" >
        <number>11</number>
       </property>
       <property name="rightMargin" >
        <number>11</number>
       </property>
       <property name="bottomMargin" >
        <number>11</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>SMS poll timer (seconds):</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KDoubleNumInput" native="1" name="kcfg_sms_pollTimeout" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QCheckBox" name="kcfg_sync_clock" >
     <property name="text" >
      <string>Ad&amp;just phone date/time at startup</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QCheckBox" name="kcfg_verbose" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>11</number>
     </property>
     <property name="topMargin" >
      <number>11</number>
     </property>
     <property name="rightMargin" >
      <number>11</number>
     </property>
     <property name="bottomMargin" >
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1_2" >
       <property name="text" >
        <string>SMS Center</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" native="1" name="kcfg_smscenter" />
     </item>
     <item>
      <widget class="KPushButton" native="1" name="b_smscenter_choose" />
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="QCheckBox" name="kcfg_calendar" >
     <property name="text" >
      <string>Calendar &amp;Support</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2" >
    <widget class="QCheckBox" name="kcfg_maildir" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>E&amp;xport SMS to MailDir</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>120</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="2" >
    <widget class="KUrlRequester" native="1" name="kcfg_maildir_path" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" >
    <widget class="QLabel" name="textLabel1_3" >
     <property name="text" >
      <string>MailDir Path</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QWidget</extends>
   <header>klineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QWidget</extends>
   <header>kpushbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >knuminput.h</include>
  <include location="local" >klineedit.h</include>
  <include location="local" >kpushbutton.h</include>
  <include location="local" >kurlrequester.h</include>
  <include location="local" >klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
